oam/install_scripts/mcs-savebrm.py.in

```
@@ -316,62 +318,61 @@ def get_save_brm_path_prefix(a_mcs_config_root):
     return get_save_brm_dir_path(a_mcs_config_root) + '/' + BRM_BACKUP_PATH_PART.format(epoch_prefix)
 
 
-def call_save_brm(path):
-    """Calls save_brm first and then tries to call it with local path. 
+def call_executable_with_params_and_result(executable: str, result: str) -> Optional[str]:
+    """Calls executable and return optional result
 
-    :param file_path: xml config XML root
-    :rtype: None
+    :param executable: executable to call
+    :param result: result to return
+    :rtype: Optional[str]
     """
-    savebrm_cmd = SAVEBRM + ' ' + path
     try:
-        subprocess.check_call(savebrm_cmd, shell=True)
+        subprocess.check_call(executable, shell=True)
     except subprocess.CalledProcessError as exc:
-        logging.error('The call to {} exits with {}.'.format(savebrm_cmd, exc.returncode))
+        logging.error('The call to {} exits with {}.'.format(executable, exc.returncode))
         return None
```

Nit.
Seems to be better to raise error with specific message instead of returning None.

---

IMHO it is better return None to higher level instead of using try-catch.

---

it's nit but 
EAFP is more common and idiomatic in Python for several reasons.